.github/workflows/ci.yml

```diff
@@ -64,7 +64,10 @@ jobs:
 
       - name: Build
         # skip check here, since we use Checkstyle task to check the added file
```

Comment:
Jar will trigger checktyle task run, right? BTW, we could remove the unnecessary and Inaccurate comments.

Reply:
Yes, We don't use the default check task, since in the current project there are some old code doesn't meet the checkstyle rule. And use a single task to check the changed files.

Has changed the comments here, please review again.

Reply:
I recommend remove -x option, if any code could not pass, we could comment ck rule or update code style workaround. If any ck is not passed, we should not let them merge to our main branch, thoughts? @qqeasonchen

Reply:
The new added code will be checked by our checkstyle rule, if the check failed the ci will fail.
The current process will still check the code style.